packages/amplify-provider-awscloudformation/src/__tests__/initializer.test.ts

```diff
@@ -33,6 +33,10 @@ describe('run', () => {
       amplify: {
         getTags: jest.fn(),
       },
+      input: {
```

Comment:
Why do we need to modify the test to have these arguments?

Reply:
This test is checking that the pre-push transformer runs during initialization, but now it only runs during a headless init so we need to simulate that in the test